### DISCUSSION TITLE
艾尔登法环|豪华中文|V1.13.2+黄金树幽影DLC-黑焰刀魂-魅影断弦+预购特典+全DLC+存档+修改器|解压即撸

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1437] [其他]
![](https://img.2468c.com/2024/06/21/1560c96a8d008.jpg)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_b70e156adf9e40aed24c10fb352b7813586e7290.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_3aec1455923ef49f4e777c2a94dbcd0256f77eb0.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_b6c4cdb36cebdbd52b97ab6e0851b7d3e41f03b3.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_e87a3e84890ab19f8995566e62762d5f8ed39315.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_1e3dfe515c04f4071207f01d62b85a1d6b560ced.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_3e556415d1bda00d749b2166ced264bec76f06ee.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/ss_c372274833ae6e5437b952fa1979430546a43ad9.600x338.jpg?t=1700164457)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/1245620/extras/ER_Steam_Gif_616x260.gif?t=1654259241)
全新奇幻动作角色扮演游戏。

崛起，黯然失色，在恩典的指引下，挥舞 Elden Ring 的力量，成为 The Lands Between 的 Elden Lord。

• 充满刺激的广阔世界
一个广阔的世界，有着各种情况的开阔领域和复杂的三维设计的巨大地下城无缝连接。当你探索时，发现未知和压倒性威胁的喜悦等待着你，带来高度的成就感。

• 创建自己的角色
除了自定义角色的外观外，您还可以自由组合您装备的武器、盔甲和魔法。您可以根据自己的游戏风格发展自己的角色，例如增加肌肉力量以成为强大的战士，或掌握魔法。

• 从神话中诞生的史诗剧
一个多层次的故事以片段的形式讲述。一部史诗般的戏剧，其中人物的各种思想在土地之间相交。

 
关于中文选择-第一次进入游戏会有语言选择的 注意看清楚了.
![](https://img.2468c.com/2024/06/21/50d1b710165ae.png)
第一种办法:如果大意错过了选择中文,可以到存档位置

C:\Users\用户名\AppData\Roaming\EldenRing

删除存档重新进入游戏 再次选择语言.

 
第二种办法:打开GAME文件夹目录下的steam_emu.ini文件,把Language=english改成Language=schinese

 
第三种办法:如果还不会…下载这个文件 解压覆盖到游戏GAME文件夹里面就行了

中文设置补丁[点我](https://download.piclabo.xyz/f/zEfo/Game.rar)

链接: https://pan.baidu.com/s/1NA7qFx1p1kBEHx9r2G5g7Q

提取码: x9ex
解压密码:laoquzhang.com
—

通关存档-方便第一时间玩DLC:[下载点我](https://download.piclabo.xyz/f/2nib/%E4%B8%BA%E9%BB%84%E9%87%91%E6%A0%91DLC%E5%87%86%E5%A4%87%E7%9A%84%E5%AD%98%E6%A1%A3-%E6%96%B9%E4%BE%BF%E7%AC%AC%E4%B8%80%E6%97%B6%E9%97%B4%E7%8E%A9DLC.rar)

风灵月影修改器:[下载点我](https://download.piclabo.xyz/f/ZVu9/Elden.Ring.Shadow.of.the.Erdtree.v1.02-v1.12.Plus.35.Trainer.rar)

**Comment by bin-lin2** [comment]
绳网大了还真是什么匠都有

**Comment by lllliufeng** [comment]
老司机带带我

**Comment by zifeiyu1111** [comment]
血源诅咒为什么还不上PC？

**Comment by kallen1314** [comment]
相当牛逼啊您

**Comment by xyz114** [comment]
想起fairy把绳网上的盗版电影换成高清普法视频……

**Comment by Cataliaer** [comment]
这怎么也有？
